compiler/ballerina-parser/src/test/java/io/ballerinalang/compiler/parser/test/syntax/types/ErrorTypeTest.java

```diff
@@ -73,7 +73,7 @@ public void testInValidErrorTypeMissingErrorTypeParams() {
 
     @Test
     public void testInValidErrorTypeExtraGt() {
-        test("error<NO_MATCHING_OBJECT>> a;", "error-type/error_type_assert_02.json");
+        test("error<NO_MATCHING_OBJECT>> a;", "error-type/error_type_assert_10.json");
```

Comment:
`error_type_assert_02.json` is also used in another test case as well. So I added `10.json` here which is identical to `02.json`